assets/styles/app.scss

```diff
@@ -1,9 +1,432 @@
+
+                .label {
+                    bottom: 10px;
+                    left: 10px;
+                    .info > div {
+                        left: 20px;
+                        opacity: 0;
+                    }
+                }
+            }
+
+            .shadow {
+                position: absolute;
+                bottom: 0px;
+                left: 0px;
+                right: 0px;
+                height: 120px;
+                transition: 0.5s cubic-bezier(0.05, 0.61, 0.41, 0.95);
+            }
+
+            .label {
+                display: flex;
+                position: absolute;
+                right: 0px;
+                height: 40px;
+                transition: 0.5s cubic-bezier(0.05, 0.61, 0.41, 0.95);
+
+                .icon {
+                    display: flex;
+                    flex-direction: row;
+                    justify-content: center;
+                    align-items: center;
+                    min-width: 40px;
+                    max-width: 40px;
+                    height: 40px;
+                    border-radius: 100%;
+                    background-color: white;
+                    color: var(--defaultBackground);
+                }
+
+                .info {
+                    display: flex;
+                    flex-direction: column;
+                    justify-content: center;
+                    margin-left: 10px;
+                    color: white;
+                    white-space: pre;
+
+                    > div {
+                        position: relative;
+                        transition: 0.5s cubic-bezier(0.05, 0.61, 0.41, 0.95),
+                            opacity 0.5s ease-out;
+                    }
+
+                    .main {
+                        font-weight: bold;
+                        font-size: 1.2rem;
+                    }
+
+                    .sub {
+                        transition-delay: 0.1s;
+                    }
+                }
+            }
+        }
+    }
+}
+
+.bullet-navigation {
+    display: flex;
+    justify-content: center;
+    margin-top: 20px;
+}
+
+.bullet {
+    width: 11px;
+    height: 11px;
+    margin: 0 5px;
+    background-color: #ccc;
+    border-radius: 50%;
+    cursor: pointer;
+    transition: background-color 0.3s;
+}
+
+.bullet.active {
+    background-color: #333;
+}
+@import "tailwindcss/base";
+@import "tailwindcss/components";
+@import "tailwindcss/utilities";
+@import url("https://fonts.googleapis.com/css2?family=Lora:ital,wght@0,400;0,500;0,600;0,700;1,400;1,500;1,600;1,700&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Hind+Madurai:wght@300;400;500;600;700&display=swap");
+
+
+// Font Var
+$titleFont: "Montserrat";
+$subtitleFont: "Lora";
+$contentFont: "Hind Madurai";
```

Comment:
I guess writing twice your code is safer right ?

Reply:
Thumb in the air